src/bundles/peer-locations.js

```diff
@@ -234,6 +234,10 @@ class PeerLocationResolver {
   async findLocations (gatewayUrls, peers) {
     const res = {}
 
+    // Normalize Gateway URLS:
+    // switch localhost to raw IP to avoid subdomain redirect AND avoid Chrome forcing https:// on such redirect
+    gatewayUrls = (Array.isArray(gatewayUrls) ? gatewayUrls : [gatewayUrls]).map(url => url.replace(/localhost:(\d+)/, '127.0.0.1:$1'))
```

Comment:
:information_source: At some point Chrome started normalizing HTTP 301 to use `https://` instead of `http://`, even if the origin is `localhost`.  This broke Flags on Peers screen, but we did not notice because most of flags were locally cached in IndexDB and was not immediately obvious it stopped working.

This is a localized fix for flags on Peers screen but may be useful to know if this bug surfaces elsewhere.